.github/actions/windows/prepare-build/action.yml

```diff
@@ -31,7 +31,7 @@ runs:
 
     - name: Setup PHP SDK
       id: setup-php
-      uses: php/setup-php-sdk@v0.8
+      uses: alcaeus/setup-php-sdk@fix-old-php-install
```

Comment:
Checked the diff: https://github.com/php/setup-php-sdk/compare/main...alcaeus:setup-php-sdk:fix-old-php-install

Is it something you will contribute back to the project?

Reply:
I always do: https://github.com/php/setup-php-sdk/pull/2